dist/activate

```diff
@@ -0,0 +1,171 @@
+# Copyright 2020 Google LLC
```

Comment:
nit: `dist` usually is a generated directory, and in this case it is the source directory. this might be confusing to others

Reply:
do you have a suggestion for a better name?
I understand dist as distribute and these are the files that will get distributed in the kctf.tar.gz, that's why I chose the name.

Reply:
maybe sdk?

Reply:
is it an sdk though? I.e. you don't use this to develop software, you use it for deployment.
Maybe it's a software deployment kit, which we could abbreviate as "sdk" ;)
I mean, in the end it doesn't matter too much since the users don't see this.

Reply:
maybe client? idk :P it's just a styling thing

Reply:
let's decide later and change in a future PR?

Reply:
yea :) - the nits is my abbreviation for "let's not bikesched this, but in case you agree, what about"